src/app.ts

```diff
@@ -0,0 +1,28 @@
+// Disable import first to allow dotenv configuration to happen before any imports
+/* eslint-disable import/first */
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { log } from './lib/log';
+import { buildServer } from './server';
+
+const APPLICATION_HOST: string = process.env.APPLICATION_HOST as string;
+const APPLICATION_PORT = Number(process.env.APPLICATION_PORT);
```

Comment:
```suggestion
const APPLICATION_PORT: number = Number(process.env.APPLICATION_PORT) | 3001;
```

Reply:
Ei implisiittisiä conffeja koodin sekaan pliidepliis

Reply:
jos halutaan defaultticonffeja niin solis sitten joku .env.dev tms.
Toki pitäis koodissa tarkastaa, että conffit on validit ([kts.](https://github.com/Tampereen-Urheilusukeltajat/Blenderi-Backend/pull/2#discussion_r967792326))

Luettavvaa: [https://12factor.net/config](https://12factor.net/config)

Reply:
Ok. .env tiedoston lisääminen riitti ongelman ratkaisemiseksi.

Reply:
Nooh, voidaan käydä siitä vielä jatkokeskustelua tulevaisuudessa :D

Reply:
Viedään tää nyt näin